Common/Securities/Cash.cs

```diff
@@ -224,8 +224,15 @@ SecurityChanges changes
                     }
                     var marketHoursDbEntry = marketHoursDatabase.GetEntry(symbol.ID.Market, symbol.Value, symbol.ID.SecurityType);
                     var exchangeHours = marketHoursDbEntry.ExchangeHours;
+
+                    // we currently do not have quote data for Crypto in backtesting, so we subscribe to trades instead of quotes
+                    var objectType = minimumResolution == Resolution.Tick
+                        ? typeof(Tick)
+                        : (securityType == SecurityType.Crypto ? typeof(TradeBar) : typeof(QuoteBar));
+                    var tickType = securityType == SecurityType.Crypto ? TickType.Trade : TickType.Quote;
+
                     // set this as an internal feed so that the data doesn't get sent into the algorithm's OnData events
-                    var config = subscriptions.Add(objectType, TickType.Quote, symbol, minimumResolution, marketHoursDbEntry.DataTimeZone, exchangeHours.TimeZone, false, true, false, true);
+                    var config = subscriptions.Add(objectType, tickType, symbol, minimumResolution, marketHoursDbEntry.DataTimeZone, exchangeHours.TimeZone, false, true, false, true);
```

Comment:
for posterity: discussion about resolving this value from outside via subscription manager's tick type lookup -- allows the value to be controller by setup handler/algorithm if required, we'll pull the first tick type for the conversion feed here.